src/client/pages/Payment/useRedirectURL.ts

```diff
@@ -0,0 +1,31 @@
+import { useLocation } from 'react-router'
+import { useMemo, useEffect } from 'react'
+
+const QUERY_PARAMETER = 'redirect_url'
+const SESSION_STORAGE_KEY = `_hedvig_${QUERY_PARAMETER}`
+
+export const useRedirectURL = () => {
```

Comment:
Let's explain why we're using localStorage as backup

Reply:
true that :)

I'll write here for now:

We need to remember the redirect URL even if Trustly does a redirect to sign with BankID in the middle of the flow.